Lib/test/test_descrtut.py

```diff
@@ -37,16 +37,16 @@ def merge(self, other):
 Here's the new type at work:
 
     >>> print(defaultdict)              # show our type
-    <class 'test.test_descrtut.defaultdict'>
+    defaultdict
```

Comment:
I think these tests looked better with the previous outputs.  Revert changes and change inputs instead (`>>> defaultdict` should show repr unless doctest doesn’t emulate the REPL displayhook)